requirements.txt

```diff
@@ -1,44 +1,18 @@
-Django==1.7.2
-Mako==1.0.1
-MarkupSafe==0.23
-MySQL-python==1.2.5
-PasteDeploy==1.5.2
-SQLAlchemy==1.0.5
-WebOb==1.4.1
+Django==1.8.3
+Pillow==2.9.0
+Pygments==2.0.2
 algorithmia==0.9.0
-argparse==1.3.0
-dj-static==0.0.6
-enum34==1.0.4
-flake8==2.2.5
-glob2==0.4.1
-gnureadline==6.3.3
-gunicorn==19.1.1
-ipython==2.3.1
-mccabe==0.2.1
-networkx==1.9.1
-nltk==3.0.0
-nose==1.3.4
-parse==1.6.6
-parse-type==0.3.4
-pbr==1.2.0
-pep8==1.6.2
+dj-database-url==0.3.0
+django-debug-toolbar==1.3.2
+django-geojson==2.8.1
+django-registration-redux==1.2
+djangorestframework==3.2.0
+factory-boy==2.5.2
+httpie==0.9.2
 psycopg2==2.6.1
-pyflakes==0.8.1
-pygame==1.9.1release
-pyramid==1.5.7
-pytest==2.6.4
-repoze.lru==0.6
+py==1.4.30
+pytest==2.7.2
 requests==2.7.0
 six==1.9.0
-static3==0.5.1
-stevedore==1.5.0
-termcolor==1.1.0
-translationstring==1.3
-venusian==1.0
-virtualenv==1.11.6
-virtualenv-clone==0.2.5
-virtualenvwrapper==4.6.0
-waitress==0.8.9
+sqlparse==0.1.16
 wsgiref==0.1.2
-zope.deprecation==4.1.2
-zope.interface==4.1.2
```

Comment:
Why do you have requirements for Django in your data structures repository?  What does Django have to do with this project?